### ISSUE TITLE
`list` is not recognized as a class

### ISSUE THREAD

**Issue by markshannon** [exaloop/codon#169]
```
class C(list):
    pass
```
`codon run test.py`
gives the following error message:
```
test_codon.py:1:1-33: error: class name 'list' is not defined
```

**Comment by inumanag** [comment]
`list` is a type alias for `List`, and aliases are currently not working when inheriting classes. `List` should work as an interim measure until we fix this.

**Comment by inumanag** [comment]
Fixed in #604.
